### PR TITLE
#33 Display more users feature

### PR DESCRIPTION
**Pull request by JeremyZyduck**
Increased amount of users displayed on admin page and sorted users alphabetically.

### CODE REVIEW

File: views/pages/admin/index.ejs [hunk 1]
```diff
@@ -1,63 +1,65 @@
...
+          <td class="align-middle"><%= data[i].email%></td>
+          <td class="align-middle"><%= data[i].role%></td>
+          <td class="align-middle">
+            <a class="btn btn-secondary edit" href="../users/edit/<%=data[i].id%>">Edit</a>
+            <a
+              class="btn btn-danger delete"
+              onclick="return alert('Are You sure?')"
+              href="../users/delete/<%=data[i].id%>"
+              >Delete</a
+            >
```
Comment (dregerm1397) [inline views/pages/admin/index.ejs]: Collapse this to look like the other button above it on line 46. It is leading to some tag line issues.

**Review by dregerm1397 — COMMENTED** [review]
Looks almost perfect. Just fix the tag line issue in views/pages/admin/index.ejs.
Nice work on the test fixes as well.

File: views/pages/admin/index.ejs (hunk 1 above)
Comment (JeremyZyduck) [reply views/pages/admin/index.ejs]: It seems like whenever I try to collapse it saving just reformats it to be like this. Could it be that I messed something up when setting up linting?

**Review by dregerm1397 — APPROVED** [review]
Thanks for changing that. Looks good now.

**Review by LiamKnutson — APPROVED** [review]
Everything looks good. It appears to cover the user criteria and doesn't look to have any problems!

**Pull request merged**